lib/utils/telemetry/generatePayload.js

```diff
@@ -300,17 +300,19 @@ module.exports = ({
     payload.isConfigValid = getConfigurationValidationResult(configuration);
     payload.dashboard.orgUid = serverless && serverless.service.orgUid;
 
-    if (serverless && command === 'deploy' && isAwsProvider) {
+    if (isAwsProvider && serverless && (command === 'deploy' || command === '')) {
       const serviceName = isObject(configuration.service)
         ? configuration.service.name
         : configuration.service;
-      const accountId = serverless.getProvider('aws').accountId;
+      const accountId = serverless && serverless.getProvider('aws').accountId;
       if (serviceName && accountId) {
         payload.projectId = crypto
           .createHash('sha256')
           .update(`${serviceName}-${accountId}`)
           .digest('base64');
       }
+
+      payload.didCreateStack = Boolean(serverless && serverless.getProvider('aws').didCreateStack);
```

Comment:
Slightly related: we mentioned the stack deletion as well, and we said "we can look at `remove` commands" -> but these don't have the `projectId` property in those events. Should we add `command === 'remove'` above in the `if` too? (or maybe in a separate PR if that's cleaner)

Reply:
That's a great point - I will add that in a separate PR